boxsdk/object/event.py

```diff
@@ -0,0 +1,19 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+from boxsdk.object.base_object import BaseObject
+
+
+class Event(BaseObject):
+    """Represents a single Box event"""
+
+    _item_type = 'event'
+
+    @property
+    def get_url(self, *args):
+        """
+        Base class override.
+        Disallow this method for this subclass, should not access Event by event_id.
+        """
+        raise AttributeError("'Event' class has no method 'get_url'")
```

Comment:
Glad that you noticed this issue.

To me, this sort of thing is a code smell. If a baseclass defines a default behavior, and then a subclass is incapable of performing that behavior, chances are, the subclass should not actually inherit from that baseclass.

I believe that to be the case here. `BaseObject` is a class that is the root for all possible persistent, addressable resource types in the REST API.

Since we've never thought about non-persistent / non-addressable objects before, we didn't think about this when we first created #15.

I have an idea of what we should do here, but I'm curious for your thoughts.